<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cardapio</class>
 <widget class="QDialog" name="Cardapio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Montar Card√°pio</string>
  </property>
  <widget class="QListView" name="listView_menu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_itens">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>511</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_procurar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_procurar">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_menu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>[ Selecione Menu ]</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_menu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>811</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inserircardapio">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>50</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir Linhas
selecionadas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_apagar">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>280</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filtrar_cardapio">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_filtrar_cardapio">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>250</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
